tools/generators/migrate-converged-pkg/schema.json

```diff
@@ -20,6 +20,11 @@
     "all": {
       "type": "boolean",
       "description": "Run generator on all vNext packages"
+    },
+    "owner": {
+      "type": "string",
+      "description": "Add particular team to CODEOWNERS file",
+      "pattern": "^@[/-\\w]+$"
```

Comment:
should we support multiple names via the command line, so folks could do `@microsoft/cxe-red @sopranopillow`. It seems OK to not support from the multiple choice because it complicates it, but if someone uses the command line switch supporting multiple owners would be good.

Reply:
right, so  this is by design and to fix current issue at hand - various packages are missing owners. 

Also if particular teams have a need to add particular people, they can do that manually. This will help them highlight that CODEOWNERS has been changed and they can extend/follow rules by particular crew. aim of this automation is to follow generic approach agreed across teams. 

> should we support multiple names via the command line

that would degragate the DX ATM as there is no CLI control from which one can `select` and then add manually some other handle. maybe autocomplete control would help but not entirely sure.